### DISCUSSION TITLE
Bärndütsch

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#36] [General]
me cha bärndütsch schribe

**Comment by timweine** [comment]
First Impressionen
 Straightforward
-  direkt

**Comment by esthicodes** [comment]
- Mir haand Nüssli gha. 
- Gha mal luege.

**Comment by esthicodes** [comment]
1. Prozedurale Programmiersprachen

Eine prozedurale Sprache folgt einer Abfolge von Anweisungen oder Befehlen, um ein gewünschtes Ergebnis zu erzielen. Jede Abfolge von Schritten wird als Prozedur bezeichnet, und ein Programm, das in einer dieser Sprachen geschrieben wurde, enthält eine oder mehrere Prozeduren. Gängige Beispiele für prozedurale Sprachen sind:

    C und C++

    Java

    Pascal

    BASIC

**Reply by esthicodes** [reply]
ä prozedurali sproch fougt ere abfoug vo ahwisige und befähle um es gwünschts ergebnis z erzile. jedi abfoug vo schritte wird als prozedur bezeichnet und es program wo i eire vo dene sproche gschribe worde isch,enthautet eini oder mehreri prozedure. gänigi -bispiu vo prozeduralesproche si:

**Comment by esthicodes** [comment]
1. Sind Sie Frau Gonzales? **
``
Nein, ich bin Frau Bauer.

Nein, sie ist Frau Bauer.

Nein, Sie sind Frau Bauer.

**Comment by wapacro** [comment]
https://www.berndeutsch.ch/

**Comment by esthicodes** [comment]
Welchen Beruf gibt es wirklich?

![image](https://user-images.githubusercontent.com/78131082/200125697-0b40521a-b2fb-4f16-996d-bcd96f14b5fe.jpeg)
A. Pflanzenstreichler
B. Honiggärtner
C. Wasserbauer

![image](https://user-images.githubusercontent.com/78131082/200125660-70cc5873-0a7e-4a2f-9114-ad70a448f646.jpeg)


Antwort:
C: Wasserbauer!
Wasserbauer kümmern sich um alle Ufer, Kanäle
und Wasserwege. Sie halten sie sauber und
überwachen, ob es irgendetwas zu reparieren
gibt. Sie pflegen Dämme und Bauwerke, die sich
am Wasser befinden. Außerdem halten sie die
Wasserwege für die Schiffe sauber und beseitigen
Hindernisse wie Treibgut oder Müll.

wasserbure kümmere sich um aui ufer, kanäu und wasserwäge. sie haute se suber und überwache, obs öppis ds repariere git. sie pflege d dämm und f bouwärch wo sech am wasser befinde. usserdäm haute sie d wasserwäge für d schiff suber und besitige hindernisse wie tribguet oder ghüder.
![Uploading 200125700...]

**Comment by esthicodes** [comment]
ig has i sechs minute gschafft.
![image](https://user-images.githubusercontent.com/78131082/200527948-9d9d6f35-d077-47f7-93ac-24417d57f0dc.jpeg)

**Comment by esthicodes** [comment]
![image](https://user-images.githubusercontent.com/78131082/200782933-8690ba56-87fb-41a8-9295-61829388bf44.jpeg)

ich bin deine Grosmutter
> i bi dis grosi
